src/scss/theme-horizontal.scss

```diff
@@ -58,7 +58,11 @@
 	// Show a right arrow when sub-level nav will be displayed to the right
 	.o-hierarchical-nav__parent.o-hierarchical-nav__outside-right[aria-expanded="true"] {
 		> a i {
-			@extend %o-ft-icons-icon--arrow-right;
+			@include oFtIconsGetSvg('arrow-right', #ffffff, 8, $apply-base-styles: false);
```

Comment:
could you use oColorsGetPaletteColor for this if there isn't a color use case set for it? Maybe link-title. There should be a color use cases sass file